integrations-and-supported-tools/matplotlib/notebooks/Neptune_Matplotlib_Support.ipynb

```diff
@@ -38,7 +42,7 @@
    },
    "outputs": [],
    "source": [
-    "! pip install --upgrade --quiet neptune-client==0.9.5 matplotlib==3.2.2 plotly==4.4.1"
+    "! pip install --upgrade --quiet neptune-client matplotlib plotly"
```

Comment:
@SiddhantSadangi the requirements here should have explicit versions so that people know what works.
We'll be updating versions periodically.